source/includes/rankings.md

```diff
@@ -381,7 +381,11 @@ curl_close($ch);
 ```
 
 
-Request aggregate rankings by date and by country.
+Request aggregated rankings by date and by country. This call returns records with ranked app ID's per country and per genre/type (not other meta-data). It's ideal if you're only interested in app rank positions.
```

Comment:
Typos, fix: This call returns records with ranked app IDs per country and per genre/category (without other meta-data).